src/routes.js

```diff
@@ -105,6 +106,14 @@ export default (
             </RequireLogin>
           )}
         />
+        <Route
+          path="/discover-more-people"
+          render={() => (
+            <RequireLogin>
```

Comment:
Any reason to require login? This page can be public IMO

Reply:
It doesn't make sense for it to be public since the only way to get to this page, you have to click `Interesting People` block link for `Discover more people`. Also this page takes into account the persons following list, but I guess that doesn't matter for unlogged users